### PR TITLE
Hero chart

### PR DESCRIPTION
**Pull request by micapam**

### CODE REVIEW

File: src/components/heroWidget/heroWidget.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import {compose} from 'recompose';
+
+import withHighcharts from './../withHighcharts';
+import withHeroChart from './../withHeroChart';
+
+// render a uniquely marked up and styled custom ColumnWidget
+// might also have a ColumnWidgetLarge or ColumnWidgetMonochrome
+const HeroWidget = (props) => {
+  return (
+    <article className="chart--line" role="article">
```
Comment (elisechant) [inline src/components/heroWidget/heroWidget.js]: `chart--hero`

File: src/components/withHeroChart.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        subtitle: {
+          useHTML: true,
+          text: `<span>Last updated <time dateTime="${dateLastUpdated}">${dateLastUpdated}</time></span>`,
+        },
+        plotOptions: {
+          line: {},
+          series: { // todo
+            animation: false,
+            marker: { 
+              enabled: false, // Can't figure out a way to hide markers on lines yet show on legend :(
```
Comment (elisechant) [inline src/components/withHeroChart.js:79]: assume we opened a JIRA for this one?

File: src/components/withHeroChart.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          },
+          pointFormatter: function() {
+            const symbolChars = {
+              'circle': 9679,
+              'diamond': 9670,
+              'triangle': 9652,
+              'square': 9632,
+              'triangle-down': 9660 };
+
+            const valueFormats = {
```
Comment (elisechant) [inline src/components/withHeroChart.js]: can move this and symbolChars outside of `withHeroChart`

File: src/components/withHeroChart.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+              'percentage': function(val) { return `${val}%`; },
+              'number': function(val) { return `$${val}`; }
```
Comment (elisechant) [inline src/components/withHeroChart.js]: number is currency?

**Review by elisechant — COMMENTED** [review]
Have we tested the legend works with responsive layout?

**Comment by micapam** [comment]
Good point re. responsive, I hadn't considered it, the Highcharts rules for this are super easy it turns out !

File: src/components/withHeroChart.js (hunk 2 above)
Comment (micapam) [reply src/components/withHeroChart.js:79]: No but I have now: PD-1389

File: src/components/withHeroChart.js (hunk 3 above)
Comment (micapam) [reply src/components/withHeroChart.js]: Sure... but where to?

Comment (micapam) [reply src/components/withHeroChart.js]: withHighCharts? Or shall I make something else?

File: src/components/withHeroChart.js (hunk 2 above)
Comment (elisechant) [reply src/components/withHeroChart.js:79]: kool kool

File: src/components/withHeroChart.js (hunk 3 above)
Comment (elisechant) [reply src/components/withHeroChart.js]: no just at the top of the file outside of the main function. very slight perf boost and better SoC

Comment (micapam) [reply src/components/withHeroChart.js]: 👍

File: src/components/withHeroChart.js (hunk 4 above)
Comment (micapam) [reply src/components/withHeroChart.js]: Good point

**Pull request merged**